pkg/collector/corechecks/cluster/orchestrator/orchestrator.go

```diff
@@ -3,6 +3,7 @@
 // This product includes software developed at Datadog (https://www.datadoghq.com/).
 // Copyright 2016-2021 Datadog, Inc.
 
+//go:build kubeapiserver && orchestrator
```

Comment:
any reasons for this change?

Reply:
I will remove it manually, I think it's goland added it automatically